.cursor/skills/investigate-ci-failure/SKILL.md

```
@@ -0,0 +1,589 @@
+gh api "repos/{org}/{repo}/commits/{head_sha}/check-runs" \
+  --jq '.check_runs[] | select(.conclusion == "failure" or .conclusion == "neutral") | {id, name, conclusion, output_title: .output.title}'
+```
+
+This gives you failed Prow jobs (with GCS URLs) and failed Konflux checks
+(with check run IDs for drill-down). Route Prow failures to section 3–4,
+Konflux failures to the Konflux Failures section.
+
+### 3. Construct GCS artifact URLs
+
+From a Prow `target_url` like:
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)